incubator/couchdb/values.yaml

```diff
@@ -51,7 +51,7 @@ image:
 ## Sidecar that connects the individual Pods into a cluster
 helperImage:
   repository: kocolosk/couchdb-statefulset-assembler
-  tag: 1.2.0
+  tag: 1.3.0
```

Comment:
This should be `2.1.0`. I updated to `2.0.0` as part of #14925 which is why you don't see it here. Eventually we'll need to redo this work over on `stable` if that PR ever gets reviewed, but let's not hold that up.

Reply:
Bumped to `2.1.0`, thanks!